docs/02-app/01-building-your-application/05-optimizing/01-caching.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Caching in Next.js
+nav_title: Caching
+description: An overview of caching mechanisms in Next.js.
+---
+
+Next.js has a few caching mechanisms to improve your application's performance. This page provides an overview of the different mechanisms, how they affect each other, and how you can use them.
+
+Please note that some are handled automatically, while others can be manually configured. Caching is also split between rendering output and fetched data, client and server environments.
+
+| Mechanism                | Purpose                                                                                                                                    | What is cached?                                                        | Where is it cached?               | Duration                                                                                                           | APIs                                                                                   |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------- |
+| Static Rendering         | Reduce the rendering cost of content that doesn’t change. All routes are statically rendered by default unless they use dynamic functions. | Pre-rendered Client Components, and Rendered Server Components Payload | Server (CDN)                      | Persistent or revalidate lifetime                                                                                  | Route Segment Config, and Dynamic Functions                                            |
+| Client-side Router Cache | Improve navigation performance. Stores RSC payloads, previously fetched and soon to be fetched (prefetch)                                  | Rendered Server Component Payload                                      | Client (in-memory)                | Single Session (cleared by hard navigation or refresh), or 30s for dynamic routes, and 5 minutes for static routes | `<Link>`, `router.prefetch`, `router.refresh`, `revalidatePath`, `revalidateTag`       |
```

Comment:
in the case of `30s for dynamic routes` it should be noted that it is `30s` since the last access to the page and it is reset each time the page is accessed again.